tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp

```diff
@@ -236,20 +239,394 @@ TEST (nnstreamer_nnfw_runtime_raw_functions, invoke)
   ret = sp->invoke_NN (&prop, &data, NULL, &output);
   EXPECT_NE (ret, 0);
 
-  *((float *)input.data) = 10.0;
+  *((float *) input.data) = 10.0;
   ret = sp->invoke_NN (&prop, &data, &input, &output);
   EXPECT_EQ (ret, 0);
-  EXPECT_EQ (*((float *)output.data), 12.0);
+  EXPECT_EQ (*((float *) output.data), 12.0);
 
-  *((float *)input.data) = 1.0;
+  *((float *) input.data) = 1.0;
   ret = sp->invoke_NN (&prop, &data, &input, &output);
   EXPECT_EQ (ret, 0);
-  EXPECT_EQ (*((float *)output.data), 3.0);
+  EXPECT_EQ (*((float *) output.data), 3.0);
 
   sp->close (&prop, &data);
   g_free (model_file);
 }
 
+
+/**
+ * @brief Test nnfw subplugin with unsuccessful invoke (single ML-API)
```

Comment:
Is this really testing uncessessful invoke?

Reply:
Thank you for comment. It's my mistake.
It's successful invoke. I will modify it.